### ISSUE TITLE
Is this method generalizable?

### ISSUE THREAD

**Issue by Fanqyu** [huang-yh/SelfOcc#15]
Hi, thanks for your great work.
As far as I know, NeuS is optimized per scene and is not generalizable. Thus, I wonder if the NeuS used in your work is adapted to a generalizable form. If yes, could you explain the implementation briefly? 
It could help me save a considerable amount of time. Looking forward to your reply.

**Comment by huang-yh** [comment]
Yes, SelfOcc is generalizable to novel scenes. At the core of the implementation is a feed-forward TPV encoder self-supervised by depth reprojection loss and neural rendering loss.
